config/wpcalypso.json

```diff
@@ -129,6 +129,7 @@
 		"recommend-plugins": true,
 		"republicize": true,
 		"rubberband-scroll-disable": false,
+		"rum-tracking/logstash": true,
```

Comment:
When is `config/wpcalypso.json` used?

Reply:
When deployed to http://wpcalypso.wordpress.com/